dbos/_context.py

```diff
@@ -202,25 +202,25 @@ def set_authentication(
 ##############################################################
 
 
-dbos_context_var: ContextVar[Optional[DBOSContext]] = ContextVar(
+_dbos_context_var: ContextVar[Optional[DBOSContext]] = ContextVar(
     "dbos_context", default=None
 )
 
 
-def set_local_dbos_context(ctx: Optional[DBOSContext]) -> None:
-    dbos_context_var.set(ctx)
+def _set_local_dbos_context(ctx: Optional[DBOSContext]) -> None:
```

Comment:
Why are these underscored in the underscored file?

Reply:
Because they are only used in the local file